src/sentry/snuba/metrics/query_builder.py

```diff
@@ -440,24 +450,21 @@ def _build_queries(self, query_definition):
                     list(metric_ids_set),
                 ),
             ]
-            orderby = self._build_orderby(query_definition)
+            orderby = self._build_orderby()
 
-            queries_dict[entity] = self._build_totals_and_series_queries(
+            queries_dict[entity] = self.__build_totals_and_series_queries(
                 entity=entity,
                 select=select,
                 where=where + where_for_entity,
                 groupby=groupby,
                 orderby=orderby,
-                limit=query_definition.limit,
-                offset=query_definition.offset,
-                rollup=query_definition.rollup,
-                intervals_len=len(list(get_intervals(query_definition))),
+                limit=self._query_definition.limit,
+                offset=self._query_definition.offset,
+                rollup=self._query_definition.rollup,
```

Comment:
those params can now probably go away, since query_definition is on the instance